skyportal/schema.py

```diff
@@ -128,6 +144,368 @@ def add_schema(schema_class_name, exclude=[], add_to_model=False):
             add_schema(f'{schema_class_name}NoID', exclude=['created_at', 'id', 'modified'])
 
 
+class PhotBaseFlexible(object):
```

Comment:
Can we add a comment or docstring here & below to explain what these different classes are used for? e.g. which are used for generating the docs and which are used for validating data

Reply:
Fixed in 9fe649d